README.adoc

```diff
@@ -36,3 +36,13 @@ They're specifically chosen to be easier for first time contributors.
 It solves all known pitfalls that can disrupt your development.
 It also describes all guidelines, tips and tricks.
 If you want your pull requests (or patches) to be merged into master, please respect those guidelines.
+
+=== Definition of Done
+
+To consider any individual ticket "Done", following requirements must be satisfied:
+
+  . Every change must go through PR; source code of both the feature/bugfix and its tests have been reviewed.
```

Comment:
In the current state of affairs, I tend to violate this rule every now and then, with trivial documentation changes (to avoid PR work overhead), and less-trivial, urgent bugfixes (because not sure who will review it quickly enough). Features do go in a PR these days.
That being said, I can stop doing that, let's discuss on irc.

Reply:
Yes, let's talk - trivial changes of documentation might be ok, for other changes I believe at least one of me, Jirka or Lukas will be always around, if not and the fix is urgent, let's make an exception. But in general, I am inclined to go through PRs in 99% of cases - it's nice opportunity to look at the code with another pair of eyes, to assess test coverage, etc.